internal/handlers/msg_service_router_test.go

```diff
@@ -1014,7 +1014,7 @@ func TestRewardsProgramStart(t *testing.T) {
 		NewEvent(rewardtypes.EventTypeRewardProgramCreated,
 			NewAttribute(rewardtypes.AttributeKeyRewardProgramID, "1")),
 		NewEvent(sdk.EventTypeMessage,
-			NewAttribute(sdk.AttributeKeyAction, rewardtypes.TypeMsgCreateRewardProgramRequest)),
+			NewAttribute(sdk.AttributeKeyAction, sdk.MsgTypeURL(&rewardtypes.MsgCreateRewardProgramRequest{}))),
```

Comment:
I don't believe this should be changed.  Currently, the `TypeMsgCreateRewardProgramRequest` variable is `create_reward_program` for the event.  Let me know what you find.

Reply:
This was due to the module.go not registering the msg router similar to other modules.  This is correct since it is now fixed.  However, I believe we have a lot of modules with the `Type*` custom event names declared but are not in use anymore.  Maybe we can clean those up?

Reply:
Agreed, I removed the Type names.